Tests/SentryTests/Integrations/SentrySystemEventsBreadcrumbsTest.swift

```diff
@@ -3,8 +3,8 @@ import XCTest
 
 class SentrySystemEventsBreadcrumbsTest: XCTestCase {
     
-    private class Fixture {
-        func getSut(scope: Scope, currentDevice: UIDevice = UIDevice.current) -> SentrySystemEventsBreadcrumbs {
+    internal class Fixture {
```

Comment:
`l`: I'm curious why did we change this to `internal`? I think it can be private.

Reply:
it should be `private`, I need a coffee, MyDevice should be `internal`, we may reuse somewhere.